Monika After Story/game/script-topics.rpy

```diff
@@ -1265,6 +1265,19 @@ label monika_portraitof:
     m 2lksdla "Actually, I don't think I should be talking about this."
     m 2hksdlb "Ahaha, sorry!"
     m 1rksdla "Just forget I said anything."
+
+    m 2hksdlb "Ahaha, sorry!{fast}{w=0.005}{nw}"
+    m 2lksdla "Actually, I don't think I should be talking about this.{fast}{w=0.005}{nw}"
+    m 1wuw "Ah...{fast}{w=0.005}{nw}"
+    m 4hub "It's funny, because I'm pretty sure that book--{fast}{w=0.005}{nw}"
+    m "Portrait of...whatever it was called...{fast}{w=0.005}{nw}"
+    m 4eua "Hey, you know that book you were reading with Yuri?{fast}{w=0.005}{nw}"
+    m 3hua "Like I said, just forget I said anything~"
+
+    python:
+        # Remove previous 14 lines from history.
+        del _history_list[-14:]
```

Comment:
I still don't think this dialogue works or is needed. What you have here was already said above. You just need to remove 3 last history entries after the `"It's funny, because I'm pretty sure that book--"` line.